content/en/Getting started/Sign In/account-recovery.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Account Recovery"
+linkTitle: "Account Recovery"
+weight: 10
+description: >
+  Learn how to recover your Cobalt account.
+---
+
+{{% pageinfo %}}
+Refer to this page if you lose access to your authenticator app or forget your password. If you have problems signing in through SAML SSO, contact your administrator for help.
+{{% /pageinfo %}}
+
+## Lost Access to Your Authenticator
+
+Users in the following roles can ask an [_Organization Owner_](../../glossary/#organization-owner) to turn off [two-factor authentication](../#two-factor-authentication) (2FA) for their account:
+
+- [_Organization Owner_](../../glossary/#organization-owner)
+- [_Organization Member_](../../glossary/#organization-member)
+- [_Pentest Team Member_](../../glossary/#pentest-team-member)
+
+Follow these steps:
+
+1. Sign in to Cobalt from the {{% sign-in %}} page in one of the following ways:
+   - With your username and password
+   - By selecting **Sign in with Google**
+
+1. On the page prompting you to enter a one-time code, select **Start account recovery process**.
```

Comment:
@molfinn The most important in this step is the action: `select **Start account recovery process**`. The first part of the sentence is only needed to understand on which page we should click the link, so there is no need to over-explain this.